package.json

```diff
@@ -66,8 +67,10 @@
     "redux-thunk": "2.3.0",
     "regenerator-runtime": "0.13.7",
     "request": "2.88.2",
+    "serve-static": "1.14.1",
     "sha256-file": "1.0.0",
     "sql.js": "pastelnetwork/sql.js#230351f",
+    "squoosh": "pastelnetwork/squoosh#dev",
```

Comment:
Can we have a commit hash instead of the branch name, similar how it's done for `sql.js` please?

Reply:
I have updated to the hash